python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_launching.py

```diff
@@ -379,3 +386,38 @@ def test_memory_and_cpu(ecs, instance, workspace, run, task_definition):
     instance.add_run_tags(run.run_id, {"ecs/memory": "999"})
     with pytest.raises(ClientError):
         instance.launch_run(run.run_id, workspace)
+
+
+def test_status(ecs, instance, workspace, run):
+    instance.launch_run(run.run_id, workspace)
+
+    # Reach into StubbedEcs and grab the task
+    # so we can modify its status. This is kind of complicated
+    # right now so it might point toward a potential refactor of
+    # our internal task data structure - maybe a dict of dicts
+    # using cluster and arn as keys - instead of a dict of lists?
+    task_arn = instance.get_run_by_id(run.run_id).tags["ecs/task_arn"]
+    task = [task for task in ecs.tasks["default"] if task["taskArn"] == task_arn][0]
+
+    running_statuses = [
```

Comment:
NIt: Maybe pull this from a constant (and use that same constant in the implementation?)